### PR TITLE
Feature/kit bundles documentation

### PR DESCRIPTION
**Pull request by malhussan**

### CODE REVIEW

File: src/commands/kit/bundle.command.ts [hunk 1]
```diff
@@ -292,6 +295,23 @@ summary: |
...
+async function applyKitDocumentationTf(
+  kitPath: string,
+  content: string | undefined,
+) {
+  const documentationTfFile = path.join(kitPath, "documentation.tf");
+  try {
+    await Deno.stat(documentationTfFile);
+  } catch (e) {
+    if (e instanceof Deno.errors.NotFound) {
+      Deno.writeTextFile(
```
Comment (nroi) [inline src/commands/kit/bundle.command.ts:307]: f: `writeTextFile` returns a Promise, so we should probably `await` (alternatively, use the sync variant: `writeTextFileSync`).

**Review by nroi — APPROVED** [review]
I left a comment about the async stuff, other than that, lgtm.

**Comment by nroi** [comment]
Johannes just announced that the CFMM website moved the domain to cloudfoundation.org. Since you're already touching the relevant files in this PR, you may consider changing the URLs while you're at it, i.e., replace 
`https://cloudfoundation.meshcloud.io/maturity-model/?selectedTool=Azure%20LZ%20Terraform%20module%20-%20ES` 
by
`https://cloudfoundation.org/maturity-model/?selectedTool=Azure%20LZ%20Terraform%20module%20-%20ES`

File: src/commands/kit/bundle.command.ts (hunk 1 above)
Comment (malhussan) [reply src/commands/kit/bundle.command.ts]: Fixed.

**Pull request merged**